doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -0,0 +1,658 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Should this section in the Serve user guide https://ray--25003.org.readthedocs.build/en/25003/serve/http-adapters.html#ray-air-modelwrapper link to this example?